Common/Charting.cs

```diff
@@ -135,6 +135,16 @@ public class Series
         /// </summary>
         public SeriesType SeriesType = SeriesType.Line;
 
+        /// <summary>
+        /// Color the series 
+        /// </summary>
+        public SeriesColor SeriesColor = SeriesColor.Black;
```

Comment:
Let's rename this to simply `Color` as it is already a member of the `Series` class